cmd/cluster-agent/api/v1/endpointschecks.go

```diff
@@ -18,6 +18,7 @@ import (
 // Install registers v1 API endpoints for endpoints checks
 func installEndpointsCheckEndpoints(r *mux.Router, sc clusteragent.ServerContext) {
 	r.HandleFunc("/endpointschecks/configs/{nodeName}", getEndpointsCheckConfigs(sc)).Methods("GET")
+	r.HandleFunc("/endpointschecks", getEndpointsChecksState(sc)).Methods("GET")
```

Comment:
```suggestion
	r.HandleFunc("/endpointschecks/configs", getEndpointsChecksConfigs(sc)).Methods("GET")
```
getEndpointsChecksState => getEndpointsChecksConfisgs since it only returns configs not states